README.md

```diff
@@ -36,7 +36,7 @@ yarn add react-select
 ```
 
 Then use it in your app:
-
+#### With React Component
```

Comment:
```suggestion
#### With React Class Component
```
Might be worth to clarify since functional components are still technically components